cmd/editscopedsk_test.go

```diff
@@ -221,3 +222,123 @@ func Test_EditScopedSkByRole(t *testing.T) {
 	require.Equal(t, us.Role, "foo")
 	require.Len(t, us.Template.Sub.Allow, 1)
 }
+
+func Test_EditScopedSkConnType(t *testing.T) {
+	ts := NewTestStore(t, "edit scope")
+	defer ts.Done(t)
+
+	_, err := ts.Store.ReadOperatorClaim()
+	require.NoError(t, err)
+
+	ts.AddAccount(t, "A")
+
+	// add the scope with a generate
+	_, _, err = ExecuteCmd(createEditSkopedSkCmd(), "--sk", "generate", "--role", "foo")
+	require.NoError(t, err)
+
+	// try to add invalid conn type
+	_, _, err = ExecuteCmd(createEditSkopedSkCmd(), "--sk", "foo", "--conn-type", "bar")
+	require.Error(t, err)
+
+	// add lower case conn type - this is prevented now, but worked in the past
+	ac, err := ts.Store.ReadAccountClaim("A")
+	require.NoError(t, err)
+	scope, ok := ac.SigningKeys.GetScope(ac.SigningKeys.Keys()[0])
+	require.True(t, ok)
+	scope.(*jwt.UserScope).Template.AllowedConnectionTypes.Add(strings.ToLower(jwt.ConnectionTypeStandard))
+	ac.SigningKeys.AddScopedSigner(scope)
```

Comment:
Was not able to see any impact of calling `AddScopedSigner()` here. But it is used [here](https://github.com/nats-io/nsc/blob/92dd1350897317c7903f79d6127ed146a14f1403/cmd/editscopedsk.go#L194) so I wanted to double check if this should be kept or removed from this test.

Reply:
I think the test is fine.